web/blackjack/src/app/game/game/game.component.ts

```diff
@@ -1,6 +1,6 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ɵɵqueryRefresh } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { interval } from 'rxjs';
+import { interval, Observable } from 'rxjs';
```

Comment:
I'm not sure where these extra imports came from, maybe they're from previous revisions?